### ISSUE TITLE
Migration not successful: Property 'content' does not exist on type 'HostedPage'.

### ISSUE THREAD

**Issue by scr4bble** [chargebee/chargebee-node#57]
### Description of the Bug

Migration from chargebee-typescript v2.x to chargebee@^3 was not fully successful.
After following the migration guide there is still one thing where typescript complains:

```
Property 'content' does not exist on type 'HostedPage'.
```

Old code:
```
const result = await this.chargebee.hosted_page
    .retrieve(hostedPageId)
    .request();
return result.hosted_page.content.subscription.id;
```
New code:
```
const result = await this.chargebee.hostedPage.retrieve(hostedPageId);
return result.hosted_page.content.subscription.id; // typescript error
```

What is the recommended action here? Could this be added into the migration guide?

### Steps to reproduce

Check the old vs new code written in the description of the bug.

### Expected Behavior

everything working after following the migration guide

### Operating System

Linux

### Language version

Node v20.17.0, typescript 5.7

### Library version

3.2.0

**Comment by cb-alish** [comment]
Hi @scr4bble, thanks for migrating to `v3` and reporting this issue. We're investigating and will release a fix soon.

**Comment by cb-alish** [comment]
Hi @scr4bble, this should be resolved in `v3.2.1`. Feel free to reopen if the issue persists!
